### ISSUE TITLE
Kid Icarus Uprising - Invisible Grind Rails

### ISSUE THREAD

**Issue by MaxWrites** [PabloMK7/citra#228]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

518f723 and earlier

### Description of Issue

Grind rails in Kid Icarus Uprising are invisible in both OpenGL and Vulkan rendering modes. This makes some sections of the game unplayable without external guidance.

### Expected Behavior

A grindrail should appear as a trail that Pit can ride on to his destination. It should be obvious when Pit has multiple rails to jump  between, or when a rail will lead into a death zone.

### Reproduction Steps

Play an on foot stage with rails, for example Chapter 13. The rail will not be visible but can be rode if approached as normal. In Chapter 13, shooting a floating switch will change the path of the rail, but this is only indicated by the color of the switch as the path of the rail cannot be seen.
![sshot1](https://github.com/user-attachments/assets/68f2642a-8bf8-44b3-a0a2-e3f01f0b0e4f)
![sshot2](https://github.com/user-attachments/assets/19592b33-131f-4b60-a3e7-eb03650facef)

### Log File

[citra_log.txt](https://github.com/user-attachments/files/16571428/citra_log.txt)

### System Configuration

CPU: Ryzen 7 5800X
GPU/Driver: Nvidia RTX 3060 Ti
RAM: 64 GB DDR4
OS: Windows 10 64 bit

**Comment by evalynethegamer** [comment]
I did also have this issue. It seems to be persistent throughout the game because I had this issue on chapter 4 and 7 as wel.

a fix would really be appreciated

**Comment by TheXbox** [comment]
> I did also have this issue. It seems to be persistent throughout the game because I had this issue on chapter 4 and 7 as wel.
> 
> a fix would really be appreciated

Agreed, As Far As My Knowledge Goes, This Issue Is One Of The Most Persistent Issues In Citra's History, It Has Been There Since 2018 With No Fix, But I'm Sure The Main Reason For That Is That No One Bothered To Look into it Yet, I'm sure It Can Turn Out To Be Such a Simple Fix, Hope This Gets Worked On Soon.

**Comment by Wunkolo** [comment]
Does this only happen with the opengl renderer for you? Have you tried it with Vulkan? Any save file that you can provide?

**Comment by TheXbox** [comment]
> Does this happen with the Hardware Shaders feature disabled? Any save file that you can provide?

It Does Happen With Hardware Shaders Disabled.
